### DISCUSSION TITLE
Data storage deals are getting reject with message "Deals from client wallet are not welcome"

### DISCUSSION THREAD

**Discussion by xloem** [filecoin-project/lotus#6870] [General Q&A] [answered]
My deals aren't reaching the chain, and further deals to miners afterward are being rejected with message "forbid <addr>" .  Does anyone know more about what this message means?

I'm running the tip of the releases branch with a patch to prevent a segfault.  It's 1.11.0 .  The tip was commit d6783736dcb119711c120033e37d4c107dcc2931 and the patch is #6787 .

I'm running the daemon with --lite, connecting to api.chain.love .

At this point all my failed deals to the miners in question are listed in lotus client list-deals as being rejected for that same forbid message as the reason.  The miners I'm looking at right now are f023530, f0152337, f021479, f070932 , f016398, f082095.  They are actually forbidding me immediately now, even on first transfer.  f082095 gives me a different variant of the error: "Deals from client wallet are not welcome".

The deals I'm making are for around 384 bytes through 4MiB, sometimes a few gigabytes.  The duration is 530 days.  The deals are made testing a backend I'm working on for git-annex to store on filecoin.  The backend is at https://github.com/xloem/git-annex-remote-lotus-cli .

**Comment by de-authority** [comment]
I think we already  have a good discussion here: https://filecoinproject.slack.com/archives/CEGN061C5/p1627037553348400

**Reply by xloem** [reply]
?  Are you saying that opening a deal for 0 FIL when the miner advertises this as their price, results in being banned?  Could anyone explain the functionality of address banning in more depth?

I'd like to know how to not get banned.

**Reply by xloem** [reply]
It's also notable that this is new with the new release for me.  In 1.10 I successfully stored data for 0 FIL, with even some of the miners that are now banning me on 1.11.

**Reply by xloem** [reply]
Or are you basically saying this is a fake internet, and I'll be abused until I stop trying to use it or escape ;P

**Reply by de-authority** [reply]
You can calm down first. A large amount of data storage is not suitable for online transmission. Patrick has given you suggestions. If you want to test, we have test methods. If you really need to store some data in the filecoin network, I think some miners will be willing to store it.

**Reply by xloem** [reply]
Could you maybe paste Patrick's suggestion in here, so I know we're on the same page?

I'm interested in a forthright storage economy where I can upload data under public rules shared equally with everyone else, like Amazon, Sia, the original Datacoin, Bitcoin SV's Metanet, or the old Qora chain.  Honestly I really think you guys _do_ provide such an economy!  I really do!

Patrick suggested the "forbid" error is not in the filecoin source but is provided by a third party tool.  Can you confirm that?  It sounds like if I keep trying miners I could find ones that don't use the tool.

But it would really be great to know what causes this error.

**Comment by xloem** [comment]
Please tell me what tool or area of the source code causes this error.

**Comment by LexLuthr** [comment]
@xloem Hi, do you still need help with this? Were you able to store your data?

**Comment by xloem** [comment, marked as answer]
I was not.  I have stopped pursuing filecoin as a storage solution for now.  Too many important details failing off-chain.
